### PR TITLE
Add permission for elasticsearchversion.

### PR DESCRIPTION
**Pull request by pritamdas99**
Signed-off-by: Pritam <pritam@appscode.com>

### CODE REVIEW

**Review by AbdullahAlShaad — APPROVED** [review]
LGTM!

File: charts/catalog-manager/templates/cluster-role.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ rules:
   - kubedb.com
   resources:
   - elasticsearches
+  - elasticsearchversions
```
Comment (ArnobKumarSaha) [inline charts/catalog-manager/templates/cluster-role.yaml:16]: remove.  `elasticsearchversions` is not in kubedb.com group

File: charts/catalog-manager/templates/cluster-role.yaml [hunk 2]
```diff
@@ -48,4 +49,12 @@ rules:
   verbs: ["get", "list", "patch"]
 - apiGroups: [""]
   resources: ["serviceaccounts"]
-  verbs: ["get", "list", "watch"]
\ No newline at end of file
+  verbs: ["get", "list", "watch"]
+- apiGroups: [ "" ]
+  resources: [ "secrets" ]
+  verbs: [ "*" ]
```
Comment (ArnobKumarSaha) [inline charts/catalog-manager/templates/cluster-role.yaml:55]: Why `*` permission is needed ? get-list-watch should be enough.

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

File: charts/catalog-manager/templates/cluster-role.yaml [hunk 3]
```diff
@@ -48,4 +48,12 @@ rules:
   verbs: ["get", "list", "patch"]
 - apiGroups: [""]
   resources: ["serviceaccounts"]
-  verbs: ["get", "list", "watch"]
\ No newline at end of file
+  verbs: ["get", "list", "watch"]
+- apiGroups: [ "" ]
+  resources: [ "secrets" ]
+  verbs: [ "get","list","watch" ]
```
Comment (ArnobKumarSaha) [inline charts/catalog-manager/templates/cluster-role.yaml:54]: Combine serviceaccounts & secrets together. As they are in same apiGroup & same level of permission.

**Review by ArnobKumarSaha — CHANGES_REQUESTED** [review]

**Review by ArnobKumarSaha — APPROVED** [review]

**Pull request merged**